File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.js
function solution(a, b) {
    var answer = 0;
    if(a%2!=0 && b%2!=0) {
        answer=a*a+b*b;
    }
    else if(a%2!=0 || b%2!=0) {
        answer=2*(a+b);
    }
    else {
        if((a-b)<0) answer = -(a-b);
        else answer= a-b;
    }
    return answer;
}

File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.py
def solution(a, b):
    answer = 0
    if a%2!=0 and b%2!=0: 
        answer=a*a+b*b
    elif a%2!=0 or b%2!=0:
        answer=2*(a+b)
    else:
        if (a-b)<0:
            answer = -(a-b)
        else: answer= a-b
    return answer